frontend/src/components/menus/NavBarTop.vue

```diff
@@ -27,21 +27,29 @@
       </v-menu>
     </template>
 
+    <template v-else>
+      <v-spacer></v-spacer>
+      <router-link class="clean-link" to="/login">
+        <span>Connexion</span>
+      </router-link>
+    </template>
   </v-toolbar>
 </template>
 
 <script>
 import firebase from 'firebase/app';
-import { mapState } from 'vuex';
+import { mapState, mapActions } from 'vuex';
 
 export default {
   computed: {
     ...mapState('config', ['appTitle', 'networkOnLine']),
-    ...mapState('authentication', ['userInfos'])
+    ...mapState('authentication', ['userInfos']),
+    ...mapActions('authentication', { userLogout: 'logout' })
   },
   methods: {
     logout() {
       firebase.auth().signOut();
+      this.userLogout;
```

Comment:
Hmmmm je pense pas que cette instruction fasse quelque chose ^^ (sans les paranhtèse la fonction n'est pas appellée) et même si elle était appellée, l'action `logout` ajoutée ne fait que reset les infos du user à null, ce qui est déja fait dans la callback authChange de firebase ici : https://github.com/Zenika/zenikast/pull/31/files#diff-0b5682b854e43f98422d3f6c46816cd2R45

Reply:
Oui, mais bizarrement ça appelle bien l'action `logout`. Une explication possible serait que `userLogout` soit un getter, ce qui a peu de sens pour une action…

Pour info, `this.userLogout` rend une promise.